### ISSUE TITLE
Failed to reproduce the experiment

### ISSUE THREAD

**Issue by jx-player** [fjc2005/DetectAnyLLM#2]
We trained a Qwen2-0.5B model without making any modifications to your code, but its performance on the validation set is slightly worse than your publicly released model on huggingface-**JiachenFu/Qwen2-0.5B-detectanyllm-detector-en**. We’d like to ask whether you made any changes to the parameters in the code during your training process or it's because the different package versions in the virtual environment?
Here are the experimental results and the experimental environment.

[merged_results.xlsx](https://github.com/user-attachments/files/22789057/merged_results.xlsx)

[our_environment.yaml](https://github.com/user-attachments/files/22789058/our_environment.yaml)

**Comment by fjc2005** [comment]
Perhaps it's because you're using multi-GPU training, which results in fewer gradient descent steps? If I recall correctly, this checkpoint was obtained from 5 epochs of single-GPU training on the ai-polish-500 dataset.
